pkg/dump/dump.go

```diff
@@ -588,7 +589,16 @@ func getProxyConfiguration(ctx context.Context, group *errgroup.Group,
 		})
 	}
 
-	if len(config.CustomEntityTypes) > 0 {
+	// If SkipCustomEntitiesWithSelectorTags is true and SelectorTags is not empty,
```

Comment:
If and when custom entities support selector tags, we'd need to revert this, correct?